Config/config.php

```diff
@@ -10,7 +10,7 @@
 $coParams = [
     'name'        => 'Custom Objects',
     'description' => 'Adds custom objects and fields features to Mautic',
-    'version'     => '0.0.21',
+    'version'     => '0.0.24',
```

Comment:
Why are we going from 21 to 24 instead of 22?

Reply:
This might be due to other open PRs
Ref: https://github.com/acquia/mc-cs-plugin-custom-objects/pull/279/files#diff-5e63046c335f797bf923ba2993ee65a0ae159454f274c1a3e463f48e190442e7